style2.css

```diff
@@ -1,261 +1,261 @@
-
-  .about-container {
-    background-image: url("lowerimage.png");
-    background-repeat: no-repeat;
-    background-position: center right;
-    background-position-y: 100px;
-    background-size: 672px 352px;
-  }
-
-  .about h1 {
-    font-family: "Crete Round", serif;
-    font-weight: 400;
-    font-size: 72px;
-    line-height: 82px;
-    letter-spacing: 0.37px;
-    color: #ff6b00;
-    padding-top: 120px;
-    margin-left: 142px;
-    margin-bottom: 24px;
-  }
-
-  .about p {
-    font-family: Inter, sans-serif;
-    font-style: normal;
-    font-weight: 400;
-    font-size: 20px;
-    line-height: 32px;
-    color: #f4f5f7;
-    margin-left: 142px;
-    width: 566px;
-    text-align: left;
-  }
-
-  .about button {
-    height: 48px;
-    background: #f55800;
-    font-family: Inter, sans-serif;
-    font-weight: 700;
-    font-size: 17px;
-    line-height: 24px;
-    margin-left: 142px;
-    text-align: center;
-    letter-spacing: 0.03em;
-    color: #fff;
-  }
-
-  #about-line {
-    margin-left: 142px;
-    margin-right: 142px;
-    border: 1px solid #fff;
-    margin-top: 209px;
-  }
-
-  .daimond {
-    width: 18px;
-    height: 18px;
-    display: inline-flex;
-    background: #ff6b00;
-    transform: rotate(-45deg);
-    margin-block-end: auto;
-  }
-
-  .rectangle {
-    width: 18px;
-    height: 18px;
-    left: 12px;
-    top: 11px;
-    background: #ff6b00;
-    margin-block-end: auto;
-    margin-right: 12px;
-    margin-top: 12px;
-  }
-
-  .circle {
-    width: 18px;
-    height: 18px;
-    left: 12px;
-    top: 11px;
-    background: #ff6b00;
-    border-radius: 150px;
-    margin-block-end: auto;
-    margin-right: 12px;
-  }
-
-  .about-discription-container {
-    display: flex;
-    margin-top: 146px;
-  }
-
-  .shapes-daimond {
-    margin-top: -90px;
-    margin-left: 20px;
-  }
-
-  .shapes-rectangle {
-    margin-left: 120px;
-  }
-
-  .shapes-circle {
-    margin-left: 120px;
-  }
-
-  .about-container h3 {
-    font-family: Inter, sans-serif;
-    font-style: normal;
-    font-weight: 400;
-    font-size: 22px;
-    line-height: 43px;
-    display: flex;
-    align-items: center;
-    text-align: center;
-    color: white;
-    margin-left: 120px;
-    justify-content: space-between;
-    margin-bottom: auto;
-  }
-
-  .circle-container {
-    margin-left: 120px;
-    display: block ruby;
-  }
-
-  .rectangle-container {
-    margin-left: 115px;
-    display: block ruby;
-  }
-
-  /* stylelint-disable-next-line no-duplicate-selectors */
-}
+/* stylelint-disable selector-type-no-unknown */
+@media (min-width: 768px) {
+  html,
+  body {
+    width: 100vw;
+  }
+
+  #drop {
+    display: none;
+  }
+
+  .name {
+    margin-left: 154px;
+  }
+
+  .desktop-version {
+    display: ruby;
+    padding-right: 156px;
+    color: white;
+  }
+
+  .desktop-version img {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin-top: 55px;
+  }
+
+  .desktop-menu {
+    margin-right: 32px;
+  }

```

Comment:
- [ ] Please add navigation links and adjust the width of the navbar in the desktop version and try to stick as much as possible to the given Figma design.
and fix it in the mobile version as well.

| Figma Design  | Your Design |
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/68796232/160947615-f01e2513-3a4f-4516-b0c5-890a915572c0.png)  | ![image](https://user-images.githubusercontent.com/68796232/160947640-598d9f91-daa8-4df0-830e-74c38a90ce1e.png)  |